roles/edpm_download_cache/tasks/packages.yml

```diff
@@ -57,6 +70,7 @@
     - download_cache
 
 - name: Download images for edpm_libvirt role
```

Comment:
Not actually an image download. The role is using `dnf` to download packages. [0]

[0]https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_libvirt/tasks/download_cache.yml

Reply:
oops, copy/paste gone wrong. Thanks Jiri.